### PR TITLE
Update Cesium to 1.75

### PR DESCRIPTION
**Pull request by platypii**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://cesium.com/blog/2020/06/01/cesiumjs-tsd/
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by platypii** [comment]
Some context: Starting with version 1.70 of Cesium, there are official typescript definition files generated by the project. It's not a typescript project, but there is some tooling to help generate the Cesium.d.ts file. I copied that definition file here.

Also starting in version 1.70 cesium has removed the global Cesium object, and is not only available through the module, so I have removed the global tests.

Additionally, there are a number of lint rules which do not match the official file. I have added the minimal set necessary to pass lint.

**Review by peterblazejewicz — COMMENTED** [review]
So shouldn't DT be just removed, instead of doing full update (and duplication).
They suggests 'you can remove @types/cesium` in the linked article.
The new version (1.75) will allow to cleanly manage update between versions:
https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package
what do you think?

**Comment by platypii** [comment]
For me it would be more convenient to be able to get just the types. The reason is that including the entirety of cesium causes my build process to be incredibly slow. Having just the types is much lighter weight. And having them in DefinitelyTyped has the benefit of being in the default @types namespace.

I held out using the last version on DefinitelyTyped (1.67) but I started running into differences with the latest API, so that's why I made the PR with updates. I would still be in favor of updating instead of removing, but not sure of the policy here. Thanks

**Comment by peterblazejewicz** [comment]
AFAIK there is no hard policy on stuff here, except of best-practises from README and overall `would fit community` approach.
If we keep DT here, it would create a split, be counter-intuitive for developers IMO (package ships with it's own types, readme/blog). Here is a tool built to verify where types are for a package, see what's the results:
https://www.typescriptlang.org/dt/search?search=cesium

I do think the official `@types/cesium` should be removed (published ones stays on NPM, a placeholder for new version is just created).
I think you could have best of both approaches, by just createing a new DT package, with no NPM direct relation, so that slimmer version can be still used as an option if you really want. That would be DT-only package.|
If you put decent 'why' in the PR, it would be probably accepted I think.

File: types/cesium/tslint.json [hunk 1]
```diff
@@ -1,6 +1,15 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
+        "interface-over-type-literal": false,
```
Comment (sheetalkamat) [inline types/cesium/tslint.json:4]: Fix all the errors.. you cannot disable any of the rules here..

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Fix linting errors

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Actually you need to remove the whole package since its shipping its own types
Please look at https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package

**Pull request closed without merging**